src/main/java/uk/gov/hmcts/reform/em/stitching/service/callback/CcdCallbackDto.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.em.stitching.service.callback;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class CcdCallbackDto {
+
+    private String propertyName;
+
+    private JsonNode caseData;
+
+    private String jwt;
+
+    public JsonNode getCaseData() {
+        return caseData;
+    }
+
+    public String getJwt() {
```

Comment:
It looks like the CcdCallbackDto is never returned (does it need to exist in that case?) but if it ever is returned do we want to @JsonIgnore this field or is it okay to return?